lib/bundle/commands/exec.rb

```diff
@@ -122,7 +119,21 @@ def run(*args, global: false, file: nil, subcommand: "")
             opt = %r{/opt/#{formula_name}([/:$])}
             next unless value.match(opt)
 
-            ENV[key] = value.gsub(opt, "/Cellar/#{formula_name}/#{formula_version}\\1")
+            cellar = "/Cellar/#{formula_name}/#{formula_version}\\1"
+
+            if key == "PATH"
```

Comment:
Feels like this should apply to any `*PATH`-type variable (e.g. `MANPATH`, `PKG_CONFIG_PATH`). Maybe test if it's colon-separated?

Reply:
Good idea, done!